kernel/arm_boards/src/board.rs

```diff
@@ -0,0 +1,8 @@
+//! Default board config file.
```

Comment:
Based on the name `board.rs`, I was expecting this file to contain a struct definition that includes all the fields that each board will need to specify, and then each board file will instantiate an instance of that struct. In fact, I think you should make that change now; otherwise it'll be difficult to standardize the content of each board config file in a maintainable way.

This module could contain a default board file too, or that could be moved to a module called `default.rs`.

Reply:
I think the makefile should explicitely select a board in the future so that they don't try to run a kernel which is inadequate for their board (it would crash easily), which is why I put a compile-time error in `board.rs`, so I'll rename `board.rs` to `unselected.rs` and put the error there.

Reply:
agreed, i like that choice.